RationaleMCP/0031/grammar.md

```diff
@@ -66,10 +66,11 @@ The _S-CHAR_ accepts Unicode other than " and \\:
 > _flat-modelica_ →\
 > &emsp; _VERSION-HEADER_\
 > &emsp; **package** _IDENT_\
-> &emsp;&emsp; ( _class-definition_ **;**\
-> &emsp;&emsp; | _global-constant_ **;**\
+> &emsp;&emsp; ( _decoration_? _class-definition_ **;**\
+> &emsp;&emsp; | _decoration_? _global-constant_ **;**\
 > &emsp;&emsp; )*\
-> &emsp;&emsp; **model** _long-class-specifier_ **;**\
+> &emsp;&emsp; _decoration_? **model** _long-class-specifier_ **;**\
+> &emsp;&emsp; ( _annotation-comment_ **;** )?
```

Comment:
Web Meeting (Hans, Gerd, Martin, Oliver)
unsure about this added line

Reply:
Not related to the decoration but to SourceLocations annotation for tool-specific table, see decision Dec 14. 2022.